app/grandchallenge/algorithms/admin.py

```diff
@@ -283,6 +283,12 @@ class AlgorithmAlgorithmInterfaceAdmin(GuardedModelAdmin):
     )
     list_filter = ("is_default", "algorithm")
 
+    def has_add_permission(self, request, obj=None):
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        return False
```

Comment:
Locked down the admin actions for the through table since I don't think we need to or should be creating or changing them in the admin. They should always be created through the UI form, which has all the necessary checks in place. 

I left deleting an option for any possible bulk deletion requests.